frontend/src/metabase/query_builder/components/expressions/Layout/Layout.module.css

```diff
@@ -0,0 +1,76 @@
+.layout {
+  display: grid;
+
+  --header-height: 48px;
+  --main-width: 688px;
+  --main-height: 280px;
+  --footer-height: 46px;
+  --sidebar-width: 424px;
+
+  grid-template-rows: min-content min-content min-content;
+  grid-template-columns:
+    minmax(var(--main-width), auto)
+    auto;
+
+  @media (max-width: 920px) {
```

Comment:
can we use standard breakouts? maybe with mantine?

Reply:
There are not enough standard ones to fill the space between the `md` and `sm` breakpoints here. I've tried solving the issue with screen-relative units and I think it works well enough! (cc @AlessioLaiso)